packages/@o3r/localization/schematics/localization-to-component/schema.json

```diff
@@ -0,0 +1,31 @@
+{
+  "$schema": "http://json-schema.org/schema",
+  "$id": "NgAddLocalizationSchematicsSchema",
+  "title": "Add Otter Localization to an existing component",
+  "description": "Add Otter Localization to an existing component",
+  "properties": {
+    "path": {
+      "type": "string",
+      "description": "Path to the component"
+    },
+    "specFilePath": {
```

Comment:
what is the use of this property?

Reply:
https://github.com/AmadeusITGroup/otter/pull/464/files#diff-c7ba2237ccab109059dc14110a7840ecc6761ba6ec20147eeaa9a421768de162R296